autogpt/llm_utils.py

```diff
@@ -12,8 +11,12 @@
 
 CFG = Config()
 
-openai.api_key = CFG.openai_api_key
-
+if CFG.llm_provider == 'openai':
+    import openai
+    openai.api_key = CFG.openai_api_key
+if CFG.llm_provider == 'anthropic':
```

Comment:
Might want to write these as an elif with an `else: raise Exception("Must configure valid llm_provider")` to make it a bit more robust to someone specifying a wrong `llm_provider` (here and elsewhere).

Reply:
Ahhh I see, the continued need to rely on openai embeddings means both need to be initialized here, and elif wouldn't work.

Reply:
Yes, so one option is to add this to the config check, e.g. similar to to the `check_openai_api_key()` 
https://github.com/anthony-sarkis/Auto-GPT/blob/b47a9331c19c63fa233dc575eb97cffbae041485/autogpt/cli.py#L81